sorting/src/dip107/ShakerSort.java

```diff
@@ -0,0 +1,30 @@
+package dip107;
+
+public class ShakerSort {
+    private static void swap(int[] data, int a, int b) {
+        int temp = data[a];
+        data[a] = data[b];
+        data[b] = temp;
+    }
+
+    public static void sort(int[] data) {
+        int l = 0;
```

Comment:
l -> left

Reply:
every sane programmer knows the meaning of "l" and "r", I feel no need in such changes!

Reply:
I am not a sane programmer, so I do not know, what l and r mean